meinberlin/templates/403_csrf.html

```diff
@@ -0,0 +1,6 @@
+{% extends "403.html" %}
+{% load i18n %}
+
+{% block error_description %}
+    <p>{% trans 'CSRF verification failed. Request aborted.' %}</p>
```

Comment:
Maybe we also say. "Please reload and try again."?

Actually that is a question for @CarolingerSeilchenspringer ! What should the text be when the csrf fails? For example if someone cannot log in or an idea form is sent, but cannot be saved because the csrf cookie is not there or not valid.

Reply:
@fuzzylogic2000 would this be correct: Die Authentifizierung Ihrer Eingabe ist fehlgeschlagen. Bitte laden Sie die Seite neu und stellen Sie sicher, dass Ihre Systemeinstellungen das Setzen eines Authentifizierungs-Cookies (Token) erlaubt. Bei Fragen können Sie sich an support@mein.berlin.de wenden.

Reply:
Die Verifizierung Ihrer Eingabe ist fehlgeschlagen. Bitte laden Sie die Seite neu und stellen Sie sicher, dass Ihre Systemeinstellungen das Setzen von Session-Cookies erlaubt. Bei Fragen können Sie sich an [support@mein.berlin.de](mailto:support@mein.berlin.de) wenden.
@CarolingerSeilchenspringer That would be ok, as the CSRD verification can also fail when submitting an idea or any other form. Are you also fine with that?

Reply:
@CarolingerSeilchenspringer Sorry, I thought we already use that email somewhere in a template, but is currently only in the settings and then used in the emails as sender. So, we would need to add that constant from the settings to a view or write a template tag just to include the email. So, we just skip that last sentence.